.travis.yml

```diff
@@ -4,12 +4,15 @@ python:
   - '3.5'
```

Comment:
Done.  I hadn't thought about it because the solution to test 3.7 when it came out was really awkward.  But now all you need to do is specify the dist and it works just fine.

Reply:
:+1: